hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/PlanExecutorTest.java

```diff
@@ -91,6 +95,11 @@ public void setUp() {
                 mock(QueryResultRegistry.class));
     }
 
+    @After
+    public void cleanUp() {
+        MockUtil.closeMocks(openMocks);
```

Comment:
for consistency you can also static import closeMocks

Reply:
Yes I thought about this too, but I wanted it to be clear we're using an internal utility for this (not a built-in method from Mockito), so that's why I left it fully declared 👍